src/components/BreadCrumbs/index.js

```diff
@@ -201,7 +201,13 @@ const BreadCrumbForEntityDetail = (
     filters = [],
   } /*: {location: Location, detail?: string, filters: Array<string>} */,
 ) => {
-  const detailToRender = !filters.length && !detail ? ' Overview' : detail;
+  let detailToRender = !filters.length && !detail ? ' Overview' : detail;
```

Comment:
A bit clearer/intuitive (positive conditions first, avoids the potential confusion of double negations).

```suggestion
  let detailToRender = filters.length > 0 || detail ? detail : 'Overview';
```